src/events.js

```diff
@@ -116,7 +118,7 @@ function getNearestItem(elements, position) {
   let minDistance = Number.POSITIVE_INFINITY;
 
   return elements
-    .filter((element) => element._display && element.inRange(position.x, position.y))
+    .filter((element) => element.options && element.options.display && element.inRange(position.x, position.y))
```

Comment:
I think we can assume we have options when we had constructed an element from those.

```suggestion
    .filter((element) => element.options.display && element.inRange(position.x, position.y))
```